### DISCUSSION TITLE
Underline and align is not kept when enter markdown

### DISCUSSION THREAD

**Discussion by CarinaSka** [facebook/lexical#3519] [Q&A] [answered]
In playground, when adding underline and align(center og right), you can see the underline and alignment added in the playground, but when you click markdown, and go back to playground, the formatting is removed.
Shouldn´t this be working as expected that it would keep it formatting?

<img width="1079" alt="Skjermbilde 2022-12-09 kl  09 56 58" src="https://user-images.githubusercontent.com/88485002/206664139-48379e4c-5f03-4010-9d85-f22b169b8579.png">
<img width="1088" alt="Skjermbilde 2022-12-09 kl  09 57 11" src="https://user-images.githubusercontent.com/88485002/206664147-d621ec88-9c2d-4ae0-9026-96eb9519ac91.png">
<img width="1103" alt="Skjermbilde 2022-12-09 kl  09 57 19" src="https://user-images.githubusercontent.com/88485002/206664153-b3d1589a-48be-49ff-91bf-60ba763c67be.png">

**Comment by thegreatercurve** [comment, marked as answer]
Neither of these are part of the [Common Markdown spec which we use for our markdown implementation](https://spec.commonmark.org/dingus/).

You can add in your own custom transformers though to extend this functionality to add in either.

**Comment by ThePatriczek** [comment]
https://discord.com/channels/953974421008293909/1087250061240573993/1087250758031904778
